app/src/main/java/com/example/internetshop/presentation/activity/fragments/BaseFragment.kt

```diff
@@ -24,7 +24,7 @@ abstract class BaseFragment : Fragment() {
         ViewModelProvider(requireActivity(), factory).get(BottomNavViewModel::class.java)
     }
 
-    abstract fun setBottomNavVisibility(): Boolean
+    open fun setBottomNavVisibility(): Boolean = true
```

Comment:
This function do nothing except return value. It doesn't `set` anything. Please rename into `getBottomNavVisibility`
Naming are very important in development. To avoid confusing situations and to keep your code clean and understandable please use proper naming.

Reply:
fixed